sr_unix/gtmci.h

```diff
@@ -39,13 +41,9 @@
  */
 #define FGNCAL_STACK ((NULL == TREF(temp_fgncal_stack)) ? fgncal_stack : TREF(temp_fgncal_stack))
 
-void	ci_restart(void);
```

Comment:
After a discussion with you, I better understand the reason for removal of GTM$CI. And think it is best captured in the commit message. GTM$CI used to invoke make_cimode() which in turn invoked ci_restart() that did some parameter massaging. All that is now done as part of gtmci() itself and so no need of the extra GTM$CI frame. Some such description would help. Thanks.

Reply:
It definitely would help have it in the commit comment. Thanks.

Reply:
Done